src/_guides/libraries/library-tour.md

```diff
@@ -1035,6 +1035,38 @@ For more information, see
 [Exceptions](/guides/language/language-tour#exceptions)
 (in the language tour) and the [Exception API reference.][Exception]
 
+### Weak references and finalizers
```

Comment:
```suggestion
### Weak references and finalizers

```

Reply:
This looks the same?

Reply:
Just adding some white space before the text of the section :)